notices.json

```diff
@@ -200,6 +200,44 @@
       }
     }
   },
+  {
+    "id": "bleve_deprecation",
+    "conditions": {
+      "audience": "sysadmin",
+      "serverVersion": ["<11.0"],
+      "displayDate": ">= 2025-10-16T00:00:00Z",
+      "serverConfig": {
+        "EnableSearching": "true",
+        "DisableDatabaseSearch": "true" }
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Transitioning from Bleve Search in Mattermost v11",
+        "description": "Experimental Bleve Search functionality is retired in Mattermost v11.0. If Bleve is enabled, search will not work until ``DisableDatabaseSearch`` is set to ``false``. See more details in this [forum post](https://forum.mattermost.com/t/transitioning-from-bleve-search-in-mattermost-v11/22982).",
+        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/server_upgrade.png",
+        "actionText": "Learn more",
+        "actionParam": "https://forum.mattermost.com/t/transitioning-from-bleve-search-in-mattermost-v11/22982"
+      }
+    }
+  },
+  {
+    "id": "certificate_deprecation",
+    "conditions": {
+      "audience": "sysadmin",
+      "serverVersion": ["<11.0"],
+      "displayDate": ">= 2025-10-16T00:00:00Z",
+      "serverConfig": { "ExperimentalSettings.ClientSideCertEnable": "true" }
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Transitioning from certificate-based authentication in Mattermost v11",
+        "description": "Experimental certificate-based authentication feature is removed in Mattermost v11.0. ``ExperimentalSettings.ClientSideCertEnable`` must be ``false`` to start the server.",
```

Comment:
Propose add link to [CBA docs](https://docs.mattermost.com/administration-guide/onboard/certificate-based-authentication.html) and be consistent with the "deprecated" language. We can also state the next sentence in the same way of recommending it be set to false before proceeding with the upgrade